hazelcast/src/main/java/com/hazelcast/spi/OperationService.java

```diff
@@ -112,19 +46,6 @@
      */
     void executeOperation(Operation op);
 
-    /**
-     * Returns true if the given operation is allowed to run on the calling thread, false otherwise.
-     * If this method returns true, then the operation can be executed using {@link #runOperationOnCallingThread(Operation)}
-     * method, otherwise {@link #executeOperation(Operation)} should be used.
-     *
-     * @param op the operation to check.
-     * @return true if the operation is allowed to run on the calling thread, false otherwise.
-     *
-     * @deprecated since 3.5 since not needed anymore.
-     */
-    @Deprecated
-    boolean isAllowedToRunOnCallingThread(Operation op);
-
```

Comment:
btw, just a reminder... `isAllowedToRunOnCallingThread()` is used by EE IMap and ICache.